index.html

```diff
@@ -33,6 +33,9 @@ <h3>Marcin zmieniłem coś w kodzie</h3>
     <p>It is my first paragraph</p>
 
     <div class="article">
+    <div class="develop">
+        BRANCH DEVELOP HAS THIS CHANGE
```

Comment:
Ok Lucas it looks really good but you need to remove that many spaces here ok? :)

Reply:
It look ok, for me.